modules/minimum-required-permissions-ipi-azure.adoc

```diff
@@ -0,0 +1,280 @@
+* `Microsoft.Network/privateDnsZones/A/delete`
+* `Microsoft.Network/privateDnsZones/SOA/read`
+* `Microsoft.Network/privateDnsZones/read`
+* `Microsoft.Network/privateDnsZones/virtualNetworkLinks/read`
+* `Microsoft.Network/privateDnsZones/virtualNetworkLinks/write`
+* `Microsoft.Network/privateDnsZones/write`
+* `Microsoft.Network/publicIPAddresses/delete`
+* `Microsoft.Network/publicIPAddresses/join/action`
+* `Microsoft.Network/publicIPAddresses/read`
+* `Microsoft.Network/publicIPAddresses/write`
+* `Microsoft.Network/virtualNetworks/join/action`
+* `Microsoft.Network/virtualNetworks/read`
+* `Microsoft.Network/virtualNetworks/subnets/join/action`
+* `Microsoft.Network/virtualNetworks/subnets/read`
+* `Microsoft.Network/virtualNetworks/subnets/write`
+* `Microsoft.Network/virtualNetworks/write`
+====
+[NOTE]
+====
+You must remove the following permissions while creating the private {product-title} cluster on Azure.
+
+* `Microsoft.Network/dnsZones/A/write`
+* `Microsoft.Network/dnsZones/CNAME/write`
+* `Microsoft.Network/dnszones/CNAME/read`
+* `Microsoft.Network/dnszones/read`
+====
+
+.Required permissions for checking health of resources
+[%collapsible]
+====
+* `Microsoft.Resourcehealth/healthevent/Activated/action`
+* `Microsoft.Resourcehealth/healthevent/InProgress/action`
+* `Microsoft.Resourcehealth/healthevent/Pending/action`
+* `Microsoft.Resourcehealth/healthevent/Resolved/action`
+* `Microsoft.Resourcehealth/healthevent/Updated/action`
+====
+
+.Required permissions for creating resource group
+[%collapsible]
+====
+* `Microsoft.Resources/subscriptions/resourceGroups/read`
+* `Microsoft.Resources/subscriptions/resourcegroups/write`
+====
+
+.Required permissions for creating resource tags
+[%collapsible]
+====
+* `Microsoft.Resources/tags/write`
+====
+
+.Required permissions for creating storage resources
+[%collapsible]
+====
+* `Microsoft.Storage/storageAccounts/blobServices/read`
+* `Microsoft.Storage/storageAccounts/blobServices/containers/write`
+* `Microsoft.Storage/storageAccounts/fileServices/read`
+* `Microsoft.Storage/storageAccounts/fileServices/shares/read`
+* `Microsoft.Storage/storageAccounts/fileServices/shares/write`
+* `Microsoft.Storage/storageAccounts/fileServices/shares/delete`
+* `Microsoft.Storage/storageAccounts/listKeys/action`
+* `Microsoft.Storage/storageAccounts/read`
+* `Microsoft.Storage/storageAccounts/write`
+====
+
+.Optional permissions for creating marketplace virtual machine resources
+[%collapsible]
+====
+* `Microsoft.MarketplaceOrdering/offertypes/publishers/offers/plans/agreements/read`
+* `Microsoft.MarketplaceOrdering/offertypes/publishers/offers/plans/agreements/write`
+====
+
+.Optional permissions for creating compute resources
+[%collapsible]
+====
+* `Microsoft.Compute/availabilitySets/write`
+* `Microsoft.Compute/images/read`
+* `Microsoft.Compute/images/write`
+* `Microsoft.Compute/images/delete`
+====
+
+.Optional permissions for enabling user-defined disk encryption set
```

Comment:
I checked again the doc, looks better to reword to `enabling user-managed encryption` to keep consistent with https://56274--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure/enabling-user-managed-encryption-azure.html, and more accurate, because not only disk encryption set need to be created, also need to create keyvault.

Reply:
Addressed